app/src/main/kotlin/at/bitfire/davdroid/ui/composable/ProgressBar.kt

```diff
@@ -0,0 +1,41 @@
+package at.bitfire.davdroid.ui.composable
+
+import androidx.compose.material3.LinearProgressIndicator
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.ProgressIndicatorDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.StrokeCap
+
+
+@Composable
+fun ProgressBar(
+    modifier: Modifier = Modifier,
+    color: androidx.compose.ui.graphics.Color = MaterialTheme.colorScheme.secondary,
+    trackColor: androidx.compose.ui.graphics.Color = ProgressIndicatorDefaults.linearTrackColor,
+    strokeCap: StrokeCap = ProgressIndicatorDefaults.LinearStrokeCap
+) {
+    LinearProgressIndicator(
+        modifier = modifier,
+        color = color,
+        trackColor = trackColor,
+        strokeCap = strokeCap
+    )
+}
+
+@Composable
+fun ProgressBar(
+    progress: () -> Float,
+    modifier: Modifier = Modifier,
+    color: androidx.compose.ui.graphics.Color = MaterialTheme.colorScheme.secondary,
```

Comment:
Why not importing `androidx.compose.ui.graphics.Color`?

Reply:
Good catch!